corehq/pillows/mappings/user_mapping.py

```diff
@@ -3,7 +3,7 @@
 
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
-USER_INDEX = es_index("hqusers_2017-03-22")
+USER_INDEX = es_index("hqusers_2017-04-24")
```

Comment:
@calellowitz you were the last one to do a user reindex so I'll ask you; is this all that's needed to trigger a reindex? (And is it the preferred way?)